src/cli/usage.js

```diff
@@ -52,7 +52,10 @@ ${bold('OPTIONS')}
     Display application changelog.
   
   --accept-self-signed-certs
-    Allows medic-conf to work with self signed certs by telling node to ignore the error 
+    Allows medic-conf to work with self signed certs by telling node to ignore the error
+  
+  --skip-dependency-check
+    Medic conf will not check that the version running is set to the same version in your package.json
```

Comment:
Nit: Mentioning `medic conf` seems unnecessary

Reply:
Instead of "your package.json" maybe "the package.json in the project's directory"?